docs/examples/rust/README.md

```diff
@@ -45,31 +37,25 @@ Note that the arguments are passed after `--`.
 
 ## keygen
 
-This tool can be used to generate key files usable by other examples.
-
-Themis supports RSA keys for some use-cases,
-but most of the features expect ECDSA keys.
+This tool can be used to generate key files usable for other examples.
+Themis supports RSA keys for some use-cases, but most features expect ECDSA keys.
```

Comment:
```suggestion
```

Well, _a-a-actually_ Themis supports RSA keys everywhere where ECDSA keys are accepted. We prefer and recommend EC, though. Examples that support only EC communicate this intent.

I’m not sure that it’s possible to mix EC and RSA in Secure Session and Secure Message, but they should be fine if used exclusively.

Currently, RustThemis does not accept RSA keys in Secure Session, but that’s a... deficiency in RustThemis alone. So I’d suggest to drop this line.

Reply:
> I’d suggest to drop this line.

agree